File: api/src/main/java/org/roylance/yaas/services/IUserDeviceService.java
package org.roylance.yaas.services;

import org.roylance.yaas.models.YaasModels;

import java.util.List;

public interface IUserDeviceService {
    void saveUserDevice(final YaasModels.UserDevice userDevice);
    List<YaasModels.UserDevice> getUserDevices(final YaasModels.UserModel user);
}

File: api/src/main/java/org/roylance/yaas/services/IUserDeviceService.kt
package org.roylance.yaas.services

import org.roylance.yaas.models.YaasModels

interface IUserDeviceService {
    fun save(request: YaasModels.UIRequest): YaasModels.UIResponse
    fun all(request: YaasModels.UIRequest): YaasModels.UIResponse
}